### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by conradbeach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Konbata Converter
+
+Konbata converts .doc and .docx files into Canvas .imscc packages.
```
Comment (jaromkmadsen) [inline README.md:3]: It's a little more specialized than this, as stated below.  It only handles files with a specific naming structure.

Comment (conradbeach) [reply README.md:3]: Yeah, it is vague. How would you recommend improving it?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+# --Canvas upload setup--
+
+# e.g. http://<canvas-url>/api
+:canvas_url: <canvas instance api url>
+
+# Found in Account >> Profile >> Generate Access Token
+:canvas_token: <canvas token>
+
+# Number found at specific course url: http://<canvas-url>/accounts/_HERE_
```
Comment (jaromkmadsen) [inline README.md]: Have you made these changes to the example as well?

Comment (conradbeach) [reply README.md]: I didn't change the example file, no.

File: README.md (hunk 1 above)
Comment (jaromkmadsen) [reply README.md:3]: "Konbata converts _and uploads_ .doc and .docx files _that have the specific naming structure (as described below)_ into Canvas .imscc packages." Or something like that that, at a glance, states that this is a specialized tool.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+:account_id: <account id>
+
+# Leave empty for DEFAULT_TIMEOUT
+:request_timeout: <request timeout seconds>
+```
+
+## Usage
+
+#### Source Files
+Add any files you wish processed into the `sources` directory inside the Konbata project directory. It expects a specific directory structure for the source files that are added to that `sources` folder.
```
Comment (jaromkmadsen) [inline README.md]: Suggested edit: "Add any files you _want to process/wish to be_ processed..."

Comment (conradbeach) [reply README.md]: Yeah, I'll change that.

**Review by jaromkmadsen — APPROVED** [review]

**Pull request merged**